### ISSUE TITLE
Setup initial web interface for observation viewing and aggregation

### ISSUE THREAD

**Issue by hellais** [ooni/backend#879]
This is about setting up a web interface for viewing aggregates of observations and displaying individual ones

**Comment by hellais** [comment]
I have setup two simple pages to start collecting internal feedback on the observation generation and analysis generation:
Observation generation: https://data.ooni.org/observations/
Analysis: https://data.ooni.org/analysis/
For example if you want to see the observations and analysis generated for the recent nepal report measurements, you can do:
https://data.ooni.org/analysis/m/20231129041111.939896_NP_webconnectivity_32105af63774eabf
https://data.ooni.org/observations/m/20231129041111.939896_NP_webconnectivity_32105af63774eabf
I would invite everyone to try it out and if you spot any noteworthy bugs, please report them inside of this issue: https://github.com/ooni/data/issues/82

**Comment by hellais** [comment]
Based on this new aggregation endpoint I made a simple dashboard to plot the aggregate observation failure cases here: https://data.ooni.org/dashboard/.

The dashboard is very rough around the edges, so things like loaders are not present and in general the interactions are not so nice as they would be on something like explorer.

What I spent most time doing was properly mapping the color scales and coming up with sane defaults for the charts and plots. The pieces of interaction with the form we will probably eventually throw away and re-implement them in explorer anyways, so they aren't very useful.

For the charting I am using the observable Plot library (https://observablehq.com/plot/) and the recently open sourced observable framework (https://observablehq.com/framework/) and it works very nicely, scaling up well also to very data heavy charts (better than vega-lite or any other charting library I have used so far). It's based on d3 and is from the same author as d3 itself.

In any case I would encourage folks to play around a bit with the dashboard as it can be a useful tool to spot any additional data quality issues with the observations being generated.
